pytorch_lightning/trainer/data_loading.py

```diff
@@ -238,7 +238,7 @@ def reset_train_dataloader(self, model: LightningModule) -> None:
             if hasattr(self.train_dataloader, 'sampler') and isinstance(self.train_dataloader.sampler, RandomSampler):
                 rank_zero_warn(
                     'You requested to overfit but enabled training dataloader shuffling.'
-                    ' We are turning it off for you.'
+                    ' We are turning the training dataloader shuffling off for you.'
```

Comment:
super nit that's a little clearer to me
```suggestion
                    ' We are turning off the training dataloader shufling for you.'
```